README.md

```diff
@@ -475,6 +479,75 @@ Example usage:
 lzma.versionNumber() // => 50020012
 ```
 
+<a name="api-parse-file-index"></a>
+#### `lzma.parseFileIndex()`
+`lzma.parseFileIndex(options, [callback])`
+
+Read .xz file metadata.
+
+`options.fileSize` needs to be an integer indicating the size of the file
+being inspected, e.g. obtained by `fs.stat()`.
+
+`options.read(count, offset, cb)` must be a function that reads `count` bytes
+from the underlying file, starting at position `offset`. If that is not
+possible, e.g. because the file does not have enough bytes, the file should
+be considered corrupt. On success, `cb` should be called with a `Buffer`
```

Comment:
Yeah, makes sense. :+1: It really won’t do anything else beside passing the error to the original callback, though.